app/views/users/index.html.erb

```diff
@@ -1,12 +1,40 @@
-<section class="users-section">
-  <ul class="users-list">
-    <% @users.each do |user| %>
-      <li>
-        Name: <%= user.name %>
-        <span class="profile-link">
-          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
-        </span>
-      </li>
-    <% end %>
-  </ul>
-</section>
+
+<div class="index_grid">
+  <section class="users-section">
+    <h2 class="friends_List">Your Friends</h2>
+    <%= render 'friends' %>
+
+  </section>
+  <section class="users-section">
+    <h2 class="friends_List">All Users</h2>
+    <ul class="users-list">
+      <% @users.each do |user| %>
+        <li>
+          Name: <%= user.name %>
+          <span class="profile-link">
+            <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+            <% if current_user.id != user.id %>
+              <%= friendship_status(user.id) %>
+            <% end %>
```

Comment:
- [ ] move the if statement to a helper method and render line 17 within the helper method instead of having the if statement directly in the view